### PR TITLE
User registration styling

### PR DESCRIPTION
**Pull request by richpeters**

### CODE REVIEW

File: static/src/js/form.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```
Comment (zatan) [inline static/src/js/form.jsx]: Can we change this path into something else to be more recognizable for what it is? Example `src/apps/accounts/registration.jsx` ?

**Comment by zatan** [comment]
1. Header `Apply for access` instead of `Apply for account`
2. Can we match top nav as per invision? https://projects.invisionapp.com/share/58G0EWTWZ6J#/screens/281122362

**Comment by zatan** [comment]
@richpeters not sure whats going on here but pages aren't loading?

**Comment by zatan** [comment]
1. http://127.0.0.1:8000/accounts/registration/ still not matching a  top nav as per https://projects.invisionapp.com/share/58G0EWTWZ6J#/screens/281122362
2. http://127.0.0.1:8000/accounts/login/ Please replace `Problems logging in?` with the `Forgo password?` it sounds like it will open report page to submit the issue instead of forgot password form.
3.  http://127.0.0.1:8000/accounts/login/  Change `Submit` button name to `Login`
4. http://127.0.0.1:8000/accounts/password-reset/ stopped working

**Comment by richpeters** [comment]
Webpack Node SASS known issue

**Comment by zatan** [comment]
1. http://127.0.0.1:8000/accounts/registration/ change Register to Login at the top.
2. http://127.0.0.1:8000/accounts/password-reset/done/ needs designing.
3. Try to reset password for your email address you will recevie email in the console url should look like this http://127.0.0.1:8000/accounts/reset/MzU/set-password/ needs designing.
4. http://127.0.0.1:8000/accounts/reset/done/ needs desinging.

**Comment by zatan** [comment]
Can I get the URL to the known issue for the webpack?

**Comment by richpeters** [comment]
https://github.com/webpack-contrib/sass-loader/issues/100 has something to do with importing files. Suggested that you use `env UV_THREADPOOL_SIZE=100` but not sure

**Pull request merged**